index.html

```diff
@@ -633,6 +633,15 @@
                         <input class="btn-input calc-trigger" type="checkbox" id="helpingHandR" />
                         <label class="btn btn-xxwide" for="helpingHandR">Helping Hand</label>
                     </div>
+                    <div class="btn-group gen-specific g5 g6">
```

Comment:
This `<div>` is misaligned and not closed properly.
These buttons should be added to [calc_bc.html](https://github.com/Zarel/honko-damagecalc/blob/master/calc_bc.html#L598) as well, for the One vs All and All vs One sections. I noticed the Terrains aren't included there either, but that can be addressed some other time.